src/components/sidecard/Sidecard.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="bcgov-pos-relative" v-bind:class="sidecardCss">
+    <div class="col-12 pt-2 bcgov-pos-relative">
+      <p />
+      <section
+        v-if="type === 'grey'"
+        id="grey-section"
+        class="bcgov-submit-container bcgov-grey-custom"
+      >
+        <h2 class="bcgov-color-heading">{{ heading }}</h2>
+        <div class="bcgov-submit-content">
+          <span>{{ content }}</span>
+        </div>
+      </section>
+      <section v-if="type === 'blue'" id="blue-section" class="bcgov-submit-container">
+        <h2 class="bcgov-heading-style">{{ heading }}</h2>
+        <div class="bcgov-submit-content">
+          <span>{{ content }}</span>
+          <a v-if="image && imageLink" v-bind:href="imageLink" target="_blank" rel="noopener noreferrer">
```

Comment:
nit: noopener noreferrer => should these be 'no-opener' or camel case?

Reply:
The rel=”noreferrer” tag is a special HTML attribute that can be added to a link tag (<a>). It prevents passing the referrer information to the target website by removing the referral info from the HTTP header.

so it has to be in the casing currently used as it is a preset html attribute

Reply:
Thank you Shreyas!